configuration/developing-community-health-applications.md

```diff
@@ -548,13 +548,14 @@ Each field that can be shown on a contact's profile is defined as an object in t
 | property | type | description | required |
 |---|---|---|---|
 | `label` | `translation key` | Label shown with the field | yes |
-| `icon` | string | The name of the icon to display beside this field, as defined through the Configuration > Icons page. | no |
-| `value` | string | The value shown for the field | yes |
-| `width` | integer | The horizontal space for the field. Common values are 12 for full width, 6 for half width, or 3 for quarter width. Default 12 | no |
-| `translate` | boolean | Whether or not to translate the value. Defaults to false. | no |
-| `appliesIf` | boolean | True if the field should be shown. | no |
-| `appliesToType` | Array of contact types | Filters the contacts for which `appliesIf` will be evaluated. For example, `['person']` or `['clinic', 'health_center']`. | no |
-| `filter` | The display filter to apply to the value, eg: `{ value: '2005-10-09', filter: 'age' }` will render as "11 years". Common filters are: `age`, `phone`, `weeksPregnant`, `relativeDate`, `relativeDay`, `fullDate`, `simpleDate`, `simpleDateTime`, `lineage`, `resourceIcon`. For the complete list of filters, and more details on what each does, check out the code in [`medic/webapp/src/js/filters` dir](https://github.com/medic/medic/tree/master/webapp/src/js/filters). | no |
+| `icon` | `string` | The name of the icon to display beside this field, as defined through the Configuration > Icons page. | no |
+| `value` | `string` | The value shown for the field | yes |
+| `filter` | `string` | The display filter to apply to the value, eg: `{ value: '2005-10-09', filter: 'age' }` will render as "11 years". Common filters are: `age`, `phone`, `weeksPregnant`, `relativeDate`, `relativeDay`, `fullDate`, `simpleDate`, `simpleDateTime`, `lineage`, `resourceIcon`. For the complete list of filters, and more details on what each does, check out the code in [`medic/webapp/src/js/filters` dir](https://github.com/medic/medic/tree/master/webapp/src/js/filters). | no |
+| `width` | `integer` | The horizontal space for the field. Common values are 12 for full width, 6 for half width, or 3 for quarter width. Default 12 | no |
+| `translate` | `boolean` | Whether or not to translate the value. Defaults to false. | no |
+| `context` | `object` | [Translation variables](https://angular-translate.github.io/docs/#/guide/06_variable-replacement) | no |
+| `appliesIf` | `boolean` | True if the field should be shown. | no |
```

Comment:
This one hasn't changed as part of this PR, but is missing the function prototype. Worth adding while we are here.